### PR TITLE
Add webhook information to api key dialog

### PR DESCRIPTION
**Pull request by reykjalin**
## Changes proposed in this Pull Request:

Adds more informative text about the webhook secret and what merchants must do to enable webhooks on their site, as per [new designs](https://www.figma.com/file/Igkr3nOx0UoGDlHrsJ4eux/%5BStripe%5D-Universal-Payment-Element-(UPE)?node-id=959%3A59586).

- Add a instructions to the API key dialog.
- Update the text in the account setup dialog.

## Testing instructions

1. Set up a fresh WooCommerce site.
2. During account setup choose to enter your API keys manually and make sure there's text that informs you of what the webhook endpoint is and where to update your Stripe settings.
3. Go to the Stripe plugin's settings.
4. Open the **Settings** tab and click **Edit account keys**.
5. Make sure there's text that informs you of what the webhook endpoint is and where to update your Stripe settings.

image

---

-   [x] Covered with tests (or have a good reason not to test in description ☝️)
-   [x] Added changelog entry (or does not apply)
-   [x] Tested on mobile (or does not apply)

**Post merge**

-   [ ] Added testing instructions to the [Release Testing Instructions wiki page](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Release-Testing-Instructions) (or does not apply)

### CODE REVIEW

File: client/settings/payment-settings/account-keys-modal.js [hunk 1]
```diff
@@ -131,11 +132,42 @@ const TestWebhookSecret = () => {
...
+					settingsLink: (
+						// eslint-disable-next-line jsx-a11y/anchor-has-content
+						<a href="https://dashboard.stripe.com/account/webhooks" />
+					),
+				},
+			} ) }
+		</p>
+	);
+};
+
```
Comment (cesarcosta99) [inline client/settings/payment-settings/account-keys-modal.js]: While adding this new text, we're reaching three instances of the same piece of code, I think now is a good moment to consider extracting that to a new component and make it reusable.

**Review by cesarcosta99 — CHANGES_REQUESTED** [review]
Thanks for addressing this, @reykjalin! Tests well! 🎉 

I left a suggestion where we can turn part of the implementation reusable.

File: client/settings/payment-settings/account-keys-modal.js (hunk 1 above)
Comment (reykjalin) [reply client/settings/payment-settings/account-keys-modal.js]: Fixed in 111a8a521ce355312f6ea89610f9c6a4ca4d7129 🙂

**Review by cesarcosta99 — APPROVED** [review]
Looks good and tests well 🚢

**Pull request merged**